### PR TITLE
Move travis ci & semantic release steps to github actions

### PR DESCRIPTION
**Pull request by georgebabey**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2  
+    - name: Setup Nodejs
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: Install dependencies
+      run: npm ci
```
Comment (binodpant) [inline .github/workflows/ci.yml:16]: I like that we use npm ci instead of npm i !!!

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          node-version: 12
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.SEMANTIC_RELEASE_NPM_TOKEN }}
+        run: npm run semantic-release
```
Comment (binodpant) [inline .github/workflows/release.yml:25]: have not looked at how we use coveralls, don't need it anymore?

**Review by binodpant — APPROVED** [review]
left minor comments, but nothing I see to not merge this

File: .github/workflows/release.yml (hunk 2 above)
Comment (georgebabey) [reply .github/workflows/release.yml:25]: Good call - When I created this PR, it wasn't wired up for this repo so I didn't include it. but it looks like @adamstankiewicz  did add it so let me add that back

**Pull request merged**